src/tsickle.ts

```diff
@@ -1136,15 +1137,18 @@ class Annotator extends ClosureRewriter {
         {options: this.tsOpts, host: this.tsHost}, this.file.fileName,
         (specifier as ts.StringLiteral).text);
     const moduleSymbol = this.typeChecker.getSymbolAtLocation(specifier);
-    this.emit(this.getForwardDeclareText(importPath, moduleSymbol, isDefaultImport));
+    this.emit(this.getForwardDeclareText(
+        importPath, moduleSymbol, /*forceTypeOnlyLoads*/ true, isDefaultImport));
```

Comment:
Super minor nit: you use different whitespace style in this comment vs the previous one.